### PR TITLE
Update Scope to Find By Release Date

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Rename scope
* Update scope to find the most recently updated
* Add specs to test scope
* Add fixtures, b/c dynamically creating phqs caused false errors.

[Finished: #107392650]

### CODE REVIEW

File: app/models/phq_assessment.rb [hunk 1]
```diff
@@ -11,10 +11,10 @@ class PhqAssessment < ActiveRecord::Base
   validates :release_date, uniqueness: { scope: :participant_id }
   validate :scores_valid
 
-  scope :latest_updated, -> { order(updated_at: :desc) }
+  scope :recenty_released, -> { order(release_date: :desc).limit(1) }
```
Comment (ericcf) [inline app/models/phq_assessment.rb]: Why limit this to one record?

Comment (michaelwehrley) [inline app/models/phq_assessment.rb]: I figured, what is the point to grab all of them if I only need one.

Comment (ericcf) [inline app/models/phq_assessment.rb]: These ActiveRecord methods are lazily evaluated: it won't touch the database until you start iterating. It seems like the scope would be more flexible without the limit.

Comment (michaelwehrley) [inline app/models/phq_assessment.rb]: but won't it start iterating over the whole collection vs just one record?

Comment (ericcf) [inline app/models/phq_assessment.rb]: I'll leave that as an exercise for the reader.

**Comment by ericcf** [comment]
:rotating_light:

File: app/models/phq_assessment.rb (hunk 1 above)
Comment (michaelwehrley) [inline app/models/phq_assessment.rb]: :smile:

**Pull request merged**